eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomeImpl.java

```diff
@@ -1,44 +1,47 @@
 package com.eternalcode.core.feature.home;
 
-import org.bukkit.Location;
-
 import java.util.Objects;
 import java.util.UUID;
+import org.bukkit.Location;
 
-class Home {
+public class HomeImpl implements Home {
```

Comment:
Public????

Reply:
Yes, because the related classes for the database and commands have been separated into other packages.

![image](https://github.com/EternalCodeTeam/EternalCore/assets/65517973/03934555-4d62-4d4c-8c68-1640266389a7)